definitions/checks/foreman_openscap/report_associations.rb

```diff
@@ -0,0 +1,29 @@
+module Checks
+  module ForemanOpenscap
+    class ReportAssociations < ForemanMaintain::Check
+      metadata do
+        label :report_associations
```

Comment:
I would suggest using `:openscap_` prefix in the label: it makes it easier when listing the checks in `fm health list` command